### PR TITLE
Add occ command to verify storage configurations

### PR DESCRIPTION
**Pull request by icewind1991**
Verifies if a configured mount is valid, additional config options can be set to test mounts with authentication backends like login credentails

usage example:
example

ref https://github.com/owncloud/core/issues/20368#issuecomment-157761007

cc @PVince81 @Xenopathic

### CODE REVIEW

**Comment by PVince81** [comment]
Great idea!

**Comment by PVince81** [comment]
:+1:

File: lib/public/files/storagenotavailableexception.php [hunk 1]
```diff
@@ -58,4 +58,23 @@ public function __construct($message = '', $code = self::STATUS_ERROR, \Exceptio
 		$l = \OC::$server->getL10N('core');
 		parent::__construct($message, $l->t('Storage not available'), $code, $previous);
 	}
+
+	public static function getStateCodeName($code) {
```
Comment (LukasReschke) [inline lib/public/files/storagenotavailableexception.php]: 10:25:17 PHPDoc is needed for OCP\Files\StorageNotAvailableException::getStateCodeName

Comment (icewind1991) [inline lib/public/files/storagenotavailableexception.php]: fixed

**Comment by PVince81** [comment]
Second review please @Xenopathic @LukasReschke @MorrisJobke @SergioBertolinSG

**Comment by LukasReschke** [comment]
Besides the already fixed remark :+1: :rocket:

**Pull request merged**